.env

```diff
@@ -1 +1,12 @@
-DB_USER=TEST
\ No newline at end of file
+# DB Config
+DB_NAME=todolistapp
+DB_USER=root
+DB_PASS=
+DB_HOST=localhost
+DB_PORT=3306
+
+
+# App Config
+PORT=5000
+SECRET_ADMIN_TOKEN=somesecretwords2
+SECRET_JWT=somesecretwords
```

Comment:
maybe we should not commit this file to the repo, since this file contain secret variable?

Reply:
I agree Kak, but should I create a .env example file without values so that others could know the.env variable template?

Reply:
make sense, okay